frontend/packages/console-dynamic-plugin-sdk/src/app/k8s/actions/k8s.ts

```diff
@@ -80,6 +84,7 @@ export const watchK8sList = (
   query: { [key: string]: string },
   k8skind: K8sModel,
   extraAction?,
+  partialMetadata = false,
```

Comment:
is this only applicable to `watchK8sList` and not `watchK8sObject`?

Reply:
Looking at the [kubernetes APIs](https://github.com/kubernetes/apimachinery/blob/v0.23.1/pkg/apis/meta/v1/types.go#L1411) a bit more I found `PartialObjectMetadata` which works for resource objects. Updated `watchK8sObject` to support `partialMetadata` as well.